tests/test_opener.py

```diff
@@ -13,6 +13,10 @@
 from fixture import mkurl, tmppath
 from fs.opener import opener
 from xrootdfs import XRootDOpener
+from xrootdfs import XRootDFile, XRootDFS
+
+from XRootD import client as xclient
```

Comment:
Not currently. Haven't been updating the imports to reflect the removal of code.